pkg/arvo/sys/vane/gall.hoon

```diff
@@ -94,7 +95,15 @@
       agent=(each agent vase)
       =beak
       marks=(map duct mark)
+      sky=(map spur path-state)
   ==
+::
++$  path-state
+  $:  bob=(unit @ud)
+      fan=((mop @ud (each noun @uvI)) lte)
```

Comment:
I think it'd be better if this were `page` instead of `noun`.

Reply:
Done, though I'm a bit unclear on whether we also need a mar/page file? At the moment the `%a` scry resolution looks like this:

```hoon
=/  res=(unit (each page @uvI))   ...
?.  ?=([~ %& *] res)  ~
``[%page !>(p.u.res)]  ::TODO mar/page?
```

Or should the last line be replaced with this instead?

```hoon
``p.u.res(q !>(q.p.u.res))
```

Reply:
The latter was decided and has been pushed.